contrib/exporters/vpclogs/vpclogs.yml.default

```diff
@@ -17,8 +17,17 @@ pipeline:
       # - internal
       # - other
   transform:
-    sa:
+    type: vpclogs
+    secadvisor:
       # exclude_started_flows: true
+  objheader:
```

Comment:
these should be part of transform step ... not seperate

Reply:
The object header fields can't be part of the transform step with the way the exporter code is currently organized. The transform operates on a single flow log entry. The header fields are generated and placed in the pipeline after we have all the flows in hand and  just before the encoding and compressing.  This is done at a certain point in the function flushFlowsToObject,

Reply:
then its' a mangle step

Reply:
Then we will need multiple mangle steps; one for the object headers and then one to compute the action Accept/Reject. OK

Reply:
Mangle step is not sufficient. We need to report the number of flow logs being stored in the object. After mangle, we go into store_buffered, which then regulates the number of objects according to various parameters. So we need to intercede after we know the number of flows actually being written to the object.

Reply:
Perhaps we need to make a new store_vpc object that parallels store_buffered.